doc/source/building/windows.rst

```diff
@@ -111,26 +114,28 @@ the following in the terminal one-by-one:
    git
 
 Each of these commands should fail as we have not provided any arguments
-to them. However a failure also implies that they are accessible on the path
+to them. However, an explicit failure from the program rather than from
+the command prompt implies that the program is accessible on the path,
 which is what we wanted to test. In turn, if an error about the command being
-not found is returned then installation of the packages didn't complete
-succesfully. If any of these are missing, you're not ready to build. Go back
+not found is returned, then installation of the packages didn't complete
+successfully. If any of these are missing, you're not ready to build. Go back
 and make sure that MSYS2 is installed correctly and has the required packages
 enabled.
 
-Now time to clone the OpenBLAS repository somewhere convenient; run the
-following line-by-line separately.
+Now it's time to clone the OpenBLAS repository somewhere convenient. Run the
+following line-by-line separately, modifying the path to your GitHub repo
+folder as appropriate.
 
 .. code:: shell
 
-   cd /c/Users/<user name>/Documents/GitHub
+   cd C:\Users\<user name>\Documents\GitHub
```

Comment:
`cd /c/` doesn't work; I think it needs the colon. Also backslash for consistency with commands below.

Reply:
This depends on where you're using MSYS2 or Git Bash or not. People who use that will know this, so the change is fine, no need to go into more detail.

Reply:
I don't understand. I was just explaining why I made the change.

Reply:
I was just saying there are definitely cases where `cd /c` works, and `cd C:` doesn't. So neither the old nor the new version will work for all users, but it's not worth going into more detail here.

Reply:
Ah, ok.